WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAddParameterViewModel.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+/// View model for `BlazeAddParameterView`
+final class BlazeAddParameterViewModel: ObservableObject {
+    @Published var key: String
+    @Published var value: String
+    @Published var hasValidationError: Bool = false
+    @Published var hasCountError: Bool = false
```

Comment:
These are not updated from outside so they should be `private(set)`

Reply:
👍🏼  https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/007848ca9359096455453f5c83191a0de2f9a60e